openspp-project/openspp-registry
# Farmer registry default UI

## **Why is this change needed?**

Need to create a new module to separate the farmer demo and farmer UI

## **How was the change implemented?**

Created a new module named spp_farmer_registry_default_ui and moved all of the ui codes of farmer demo to this new module

## **New unit tests**

None

## **Unit tests executed by the author**

None

## **How to test manually**

- Install spp_farmer_registry_default_ui module
- Check if it installed without error
- Check if the UI of Group Registry has changed.
- Check if there is a GIS View

- Install spp_farmer_registry_demo module
- Check if it installed without an error
- Check if all functionality of farmer registry demo still retains

## **Related links**

https://github.com/OpenSPP/openspp-modules/issues/567
